README.md

```diff
@@ -286,15 +288,16 @@ expr 30 \> 20 #1 (true)
 ```bash
 # Number of decimal digit/ significant figure
 echo "scale=2;2/3" | bc
-#.66
+
+$ .66
```

Comment:
should be # instead of $

Reply:
Same as above, it is following your style for returned values. where you uses the `#` as the shell promt indication, usual this would be the `$` sign.  Should I replace them all for you to the more used `$` or would you keep using the `#`?